### PR TITLE
Enable PodPreset admission plugin in local-up-cluster

### PR DESCRIPTION
**Pull request by deyuanix**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -401,7 +401,7 @@ function start_apiserver {
     fi
 
     # Admission Controllers to invoke prior to persisting objects in cluster
-    ADMISSION_CONTROL=Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount${security_admission},ResourceQuota,DefaultStorageClass,DefaultTolerationSeconds
+    ADMISSION_CONTROL=Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount${security_admission},ResourceQuota,DefaultStorageClass,DefaultTolerationSeconds,PodPreset
```
Comment (gyliu513) [inline hack/local-up-cluster.sh]: How about support this as an env var so that we do not need to update it every time?

Comment (k82cn) [inline hack/local-up-cluster.sh]: Please to make ResourceQuota as the latest one of Admission controllers.

Comment (ddysher) [inline hack/local-up-cluster.sh]: yeah, but that's another story from what this pr does...

Comment (ddysher) [inline hack/local-up-cluster.sh]: I've changed the order.

**Comment by eparis** [comment]
@k8s-bot ok to test
/release-note-none
/approve
@derekwaynecarr you seem to touch admission controllers a lot. Do we need to do anything else?

File: hack/local-up-cluster.sh (hunk 1 above)
Comment (gyliu513) [inline hack/local-up-cluster.sh]: +1, filed an issue https://github.com/kubernetes/kubernetes/issues/47385 , feel free to take it if you want.

**Comment by brendandburns** [comment]
/lgtm

/release-note-none

LGTM + release-note-none from the bulk LGTM tool

**Pull request closed without merging**